staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -866,3 +872,11 @@ type SerializedNodeConfigSource struct {
 	// +optional
 	Source v1.NodeConfigSource `json:"source,omitempty" protobuf:"bytes,1,opt,name=source"`
 }
+
+// LoggingConfig contains logging options
+type LoggingConfig struct {
```

Comment:
define this type in staging/src/k8s.io/component-base/config/v1alpha1/types.go, name it `LoggingConfiguration`, and reference it from KubeletConfiguration

Reply:
I add LoggingConfiguration to the component-base/config and change kubelet config name to Logging